CHANGELOG.unreleased.md

```diff
@@ -9,6 +9,7 @@
 
 - [SDN Controller] Allow private network creation on bond and VLAN (PR [#4682](https://github.com/vatesfr/xen-orchestra/pull/4682))
 - [XOA] Display XOA build number [#4693](https://github.com/vatesfr/xen-orchestra/issues/4693) (PR [#4694](https://github.com/vatesfr/xen-orchestra/pull/4694))
+- [Hub] [ability to create recipes](https://xen-orchestra.com/blog/devblog-5-kubernetes-clutser-on-xo/) (PR [#4695](https://github.com/vatesfr/xen-orchestra/pull/4695))
```

Comment:
Maybe also, talk about Kubernetes Cluster?
`- [Hub/Recipes] Ability to create a Kubernetes cluster`

Reply:
I agree, @pdonias what do you think ?

Reply:
Yes.